src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs

```diff
@@ -45,7 +45,7 @@
 // to distinguish one build from another. AssemblyFileVersion is specified
 // in AssemblyVersionInfo.cs so that it can be easily incremented by the
 // automated build process.
-[assembly: AssemblyVersion("1.3.1.1514")]
+[assembly: AssemblyVersion("1.3.1.1977")]
```

Comment:
I think you should send future PRs to master - and no need to commit this file, its auto generated.